addons/website_slides/controllers/mail.py

```diff
@@ -69,7 +74,13 @@ def mail_update_message(self, res_model, res_id, message, message_id, redirect=N
             rating.write({
                 'rating': float(post['rating_value'])
             })
-
-        # redirect to specified or referrer or simply channel page as fallback
-        redirect_url = redirect or (request.httprequest.referrer and request.httprequest.referrer + '#review') or '/slides/%s' % res_id
-        return werkzeug.utils.redirect(redirect_url, 302)
+        channel = request.env[res_model].browse(res_id)
+        return {
+            'message_id': message.id,
+            'message': html2plaintext(message.body),
```

Comment:
Why do we need to return the plaintext here?

Reply:
comments/rating on course are stored in db as html format, but here we need it in plaintext format to avoid HTML tags appearing in composer